Some more fixes

SAbotsup opened the pull request.

SAbotsup committed:
Update details.php

SAbotsup committed:
Update watch.php

SAbotsup committed:
Update header.php

SAbotsup committed:
Update sidenav.php

SAbotsup committed:
Update sw.js

SAbotsup committed:
Update apfc.php

SAbotsup committed:
Update watch.php

SAbotsup committed:
Update details.php

SAbotsup committed:
Update watch.php

SAbotsup committed:
Update details.php

DrBrainlessLol commented:
Are you testing these before opening pr?

SAbotsup commented:
Yes bro

SAbotsup commented:
do you know how can i fix servers issue

DrBrainlessLol commented:
> do you know how can i fix servers issue

No, i removed all hianime servers as they're a waste of time and only using vidcloud and vidstreaming

DrBrainlessLol commented:
> Yes bro

I tried it last night and still had the issue with jp titles not showing so I added null checks throughout to avoid showing broken codes and stick to English if jp titles aren't available which is kind of useless anyway cus what's the point

SAbotsup commented:
my hd-2 and hd-3 server is working but hd-1 isn't i tried every thing but i cant fix this

SAbotsup commented:
Today I also fixed the filter page. I will send a pull request, but I don't know where Paca is. He is not replying, and also fixed many other pages too.

DrBrainlessLol commented:
> Today I also fixed the filter page. I will send a pull request, but I don't know where Paca is. He is not replying, and also fixed many other pages too.

Im currently trying to switch from hianime entirely to anilist, if you can fix the jp title issues tho that'd be great!! Dm me on Discord @brainlessintellect
